Documentation/user-guides/getting-started.md

```diff
@@ -59,7 +59,7 @@ describe the targets to be monitored by Prometheus.
 
 <!-- do not change this link without verifying that the image will display correctly on https://prometheus-operator.dev -->
 
-![Prometheus Operator Architecture](/img/architecture.png)
+![Prometheus Operator Architecture](../img/architecture.png)
```

Comment:
please check the comment at L60: the location is meant to work with the website.

Reply:
Is using the github direct link is a valid option?

Reply:
I assume that it wouldn't work either.

Reply:
But then does that mean that the link present in GitHub will never work?

Reply:
yes